app/src/main/java/ch/epfl/skysync/ui/components/FlightForm.kt

```diff
@@ -0,0 +1,666 @@
+                    var expandedVehicleMenu by remember { mutableStateOf(false) }
+
+                    ExposedDropdownMenuBox(
+                        modifier =
+                            Modifier.fillMaxWidth()
+                                .clickable { expandedVehicleMenu = true }
+                                .testTag("Vehicle Menu ${listVehiclesValue.size}"),
+                        expanded = expandedVehicleMenu,
+                        onExpandedChange = { expandedVehicleMenu = !expandedVehicleMenu }) {
+                          OutlinedTextField(
+                              value = vehicle?.name ?: "Vehicle ${listVehiclesValue.size + 1}",
+                              modifier =
+                                  Modifier.menuAnchor().fillMaxWidth().padding(defaultPadding),
+                              readOnly = true,
+                              onValueChange = {},
+                              trailingIcon = {
+                                ExposedDropdownMenuDefaults.TrailingIcon(
+                                    expanded = expandedVehicleMenu)
+                              })
+
+                          DropdownMenu(
+                              expanded = expandedVehicleMenu,
+                              onDismissRequest = { expandedVehicleMenu = false }) {
+                                allVehicles.withIndex().forEach { (id, item) ->
+                                  if (!listVehiclesValue.contains(item)) {
+                                    DropdownMenuItem(
+                                        modifier =
+                                            Modifier.fillMaxWidth()
+                                                .padding(defaultPadding)
+                                                .testTag("Vehicle $id"),
+                                        onClick = {
+                                          listVehiclesValue.add(item)
+                                          expandedVehicleMenu = false
+                                          addVehicle = false
+                                        },
+                                        text = { Text(item.name) })
+                                  }
+                                }
+                              }
+                        }
+                  }
+                }
+                // Drop down menu for the balloon
+                item {
+                  val balloonTitle = "Balloon"
+                  Text(
+                      modifier = Modifier.padding(horizontal = defaultPadding),
+                      text = balloonTitle,
+                      style = MaterialTheme.typography.headlineSmall)
+                  MenuDropDown(
+                      defaultPadding = defaultPadding,
+                      title = balloonTitle,
+                      value = balloonValue,
+                      onclickMenu = { item -> balloonValue = item },
+                      items = balloons)
```

Comment:
You're missing a `showString = {it?.name ?: "No balloon selected"}` or something like that to display a user friendly message.

Reply:
I just realized I have commited this but not pushed...